test/parse_tests/test_parse_es.py

```
@@ -0,0 +1,189 @@
+                         (timedelta(weeks=25), ""))
+        self.assertEqual(extract_duration("7.5 segundos"),
+                         (timedelta(seconds=7.5), ""))
+        self.assertEqual(extract_duration("10-segundos"),
+                         (timedelta(seconds=10.0), ""))
+        self.assertEqual(extract_duration("5-minutos"),
+                         (timedelta(minutes=5), ""))
+
+    def test_non_std_units(self):
+        self.assertEqual(
+            extract_duration("1 mes"),
+            (timedelta(days=DAYS_IN_1_MONTH), ""))
```

Ruff

Undefined name `DAYS_IN_1_MONTH`

(F821)